db/migrate/20200508062602_create_organizations.rb

```diff
@@ -0,0 +1,10 @@
+class CreateOrganizations < ActiveRecord::Migration[6.0]
+  def change
+    create_table :organizations do |t|
+      t.string :name
+
```

Comment:
需再討論payment or paid 要儲存什麼？是否已付款？
```ruby
t.boolean :paid, null: false, default: false
```
Karol Moroz from www.makeprogramming.fun says:
> Always remember to set `null: false, default: false` when dealing with boolean values in SQL (regardless of dialect). Otherwise, the default value for a boolean column is NULL, and you end up with three-value boolean logic (true, false, nil) instead of the true/false you would expect.

若區分多種付款方案則要加model